### DISCUSSION TITLE
Yusuf İpek'in GüvendeKal.org yorumu

### DISCUSSION THREAD

**Discussion by islemci** [GuvendeKal/guvendekal.org#131] [Genel]
❤️ Yusuf İpek (@yusufipk), canlı yayınında projemizin küçük bir incelemesini yaptı. Daha fazla kişiye ulaşmamızı sağladığı için teşekkür etmek isterim kendisine.

Destek olmak isterseniz Youtube linki: https://youtu.be/GHHz_Pl7204

**Comment by GokturkTalha** [comment]
Kendisine teşekkürler tekrardan :)
